app/views/requirements/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :sidebar do %>
+  <h2>Edit Requirement</h2>
+  <h3>(for <%= @requirement.task.title %>)</h3>
+  <%= sidebar_button_link 'Return to Task', @requirement.task %>
+  <%= sidebar_button_link 'Return to Requirements List', requirements_path %>
```

Comment:
We can make a new issue out of this, but I don't think there's much value to requirements without the context of an event.